friends/src/App.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+import FriendsList from './components/FriendsList';
+import './App.css';
+
+class App extends Component {
+	constructor() {
```

Comment:
You can now drop the `constructor` and make `state` a `class property`, give it a try.
```suggestion
	state = {
	  data: [],
	  name: '',
	  age: '',
	  email: '',
	}
```